db/migrate/20160403055927_create_deployment_authentications.rb

```diff
@@ -0,0 +1,40 @@
+class CreateDeploymentAuthentications < ActiveRecord::Migration[5.0]
+  def change
+    create_table :container_deployment_authentications do |t|
+      t.string :name
+      t.boolean :challenge
```

Comment:
not sure about the semantics here, but does it make sense to default the booleans to false? Thats what I always try, but I see our schema has lots of booleans that default to null too